canonical-websites/www.ubuntu.com
# Whitepaper on the web

## Done
Built the robotics whitepaper.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/whitepapers/robotics](http://0.0.0.0:8001/whitepapers/robotics)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /whitepapers/robotics
- Check that looks ok
- Check on small screens
- Check it matches [the whitepaper](https://drive.google.com/file/d/1pk8j0wuPsfwmaRSrTwaU8MTH_frFX2EV/view?usp=sharing)

## Issue / Card
Fixes #4782
